src/openms/include/OpenMS/KERNEL/MSChromatogram.h

```diff
@@ -424,7 +424,7 @@ namespace OpenMS
 
       @param other a reference to the MSChromatogram to take ChromatogramPeaks from
     */
-    void mergePeaks(MSChromatogram& other);
+    void mergePeaks(MSChromatogram& other, bool add_meta=true);
```

Comment:
I would make this false as default

Reply:
document if m/z stays the same

Reply:
also document what happens to
  MSChromatogram::FloatDataArray float_array;
  MSChromatogram::StringDataArray string_array;
  MSChromatogram::IntegerDataArray int_array;

Reply:
just figured that add_meta could be misunderstood that meta data is added from the other chromatogram

Reply:
Documented.